src/GameLoop.tsx

```diff
@@ -90,6 +93,10 @@ let didInstantDrop = false;
 let leaveGroundPenalty = 0;
 const leaveGroundRate = 250;
 
+// This has trouble being used as React state due to React's asynchronous updates.
+let countdownTimeElapsed: number = 0;
```

Comment:
It's useful to say what time unit this is using.  Also, globals bad.

Reply:
Done. The globals issue is addressed in #34